docs/pages/versions/unversioned/sdk/notifications.mdx

```diff
@@ -432,23 +432,23 @@ Learn how to configure the native projects in the [installation instructions in
 
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +22]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Hi @vonovak I believe we are just doing warnings in SDK 52 and disabling this feature in Expo Go for SDK 53, correct?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/expo/expo/32427?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
fixed it